CHANGES.rst

```diff
@@ -10,6 +10,8 @@
 - Fix a bug where reflecting a table could raise a ``NoSuchTableError``
   in cases where its schema is not on the current ``search_path``
   (`Issue #64 <https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/pull/64>`_)
+- Add python 3.5 to the list of versions for integration tests.
+  (`Issue #61 <https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/pull/61>`_)
```

Comment:
It's never been clear to me whether explicitly listing supported versions is actually useful to anybody.

I guess we might as well?